code/modules/projectiles/projectile.dm

```diff
@@ -65,6 +65,9 @@
 
 	var/scatter = 0 //Chance of scattering, also maximum amount scattered. High variance.
 
+	/// Used to transfer iff_signal from gun to projectile when aim mode is used
+	var/list/has_iff = list()
```

Comment:
Has_iff is a bad name as it implies its a bool

Reply:
additionally since this is an empty list it will always be true

Reply:
changed